tests/hazmat/primitives/test_dh.py

```diff
@@ -428,6 +529,26 @@ def test_private_bytes_unsupported_encryption_type(self, backend):
                 DummyKeySerializationEncryption()
             )
 
+    @pytest.mark.skipif(
```

Comment:
This is an OpenSSL backend specific test so it needs to live in `tests/hazmat/backends/test_openssl`